src/components/control/date.jsx

```diff
@@ -63,13 +72,19 @@ const Date = forwardRef(function Date(props, forwardedRef) {
 
     validate();
     setActive(false);
-    if (!refs.input.current.validationMessage) setEditing(false);
+    if (!props.validationMessage) setEditing(false);
```

Comment:
I need to check this once it is deployed. Date control should stay in edit mode when an entered date is invalid and the user leaves the control.

Reply:
This was a previous bug, we can check into it. With the `refs.input.current.validationMessage` we actually got `null` as a String